### ISSUE TITLE
Branch to https://github.com/imagej/imagej-itk

### ISSUE THREAD

**Issue by dietzc** [Squareys/imglib-itk-experiments#1]
thanks!!

**Comment by Squareys** [comment]
The imagej-itk repository is completely different from this one. It could theoretically be a branch just containing the contents of this repository, but it would never be meant to be merged.

**Comment by dietzc** [comment]
why? in the long run this will be exactly the thing replacing the current prototype, right? maybe we have to find a better name like imagej-itk-ops then, because we will auto-generate ops.

**Comment by Squareys** [comment]
This is more of a "proof of concept", there is alot of unnecessary and unfinished code, the package and class naming is not nice (`SneakyImage`?), the project structure also needs improvement. Also, this "playground" tackles "how to share memory between imglib2 and itk" rather than touching the imagej-ops topic (see readme).

To have a stable and long-term-durable project I would recommend starting this over _inspired by_, but not _based on_ this repository to whoever will continue working on this (most likely myself as of May).

One can still have this repository as a branch in imagej-itk to have things "in one place".

**Comment by dietzc** [comment]
> One can still have this repository as a branch in imagej-itk to have things "in one place".

yep. this is my point. the other project is also more a proof of concept.
